components/atoms/Counter/index.jsx

```diff
@@ -0,0 +1,11 @@
+import styles from './styles.module.scss';
+
+const Counter = ({ quantity = 0 }) => {
+	return (
+		<>
+			<p className={styles.counter}>{quantity}</p>
+		</>
+	);
+};
```

Comment:
```suggestion
const Counter = ({ quantity = 0 }) => {
	return <p className={styles.counter}>{quantity}</p>
};
```
Podés insertar el parrafo directamente si solo va a retornar eso. Te ahorras los parentesis y los <>fragments</>